src/client/components/Inputs/Select/Indicators/Indicators.scss

```diff
@@ -46,3 +46,18 @@ div.select__clearIndicator {
     display: none;
   }
 }
+
+.select__toggleAllOption-checkbox {
```

Comment:
This is very similar to the deprecated `ButtonCheckBox`. 
The new one has a extra border for the label so it doesn't fit that well in this case:
![image](https://github.com/openforis/fra-platform/assets/41337901/42aa1b06-6263-4e04-9ccf-a73d8fa2a99a)

Should we add another version similar to the previous implementation? Maybe by passing a 'type' prop that allows to choose which checkbox.

Reply:
The one in the video is the one that's applied. And okay, sounds good 👍